HSStockChart/Classes/DrawLayerProtocol/HSDrawLayerProtocol.swift

```diff
@@ -21,22 +21,23 @@ protocol HSDrawLayerProtocol {
     
     var theme: HSTimeLineStyle { get }
     
-    func drawLine(lineWidth: CGFloat, startPoint: CGPoint, endPoint: CGPoint, strokeColor: UIColor, fillColor: UIColor, isDash: Bool, isAnimate: Bool) -> CAShapeLayer
```

Comment:
I add this "HSDrawLayerProtocol" for any view which implements this protocol can have the default function: "drawLine"，"drawTextLayer", etc. It's kind of "protocol oriented programming" . I think it's not a good way to make an extension for "CAShapeLayer" and adding static function in it.